GetStream/stream-chat-android
# [3326] Fix disabling commands

### 🎯 Goal

Disabled commands are reenabled when after selecting and clearing attachments.

### 🎨 UI Changes

<table>
<thead>
<tr>
<th>Before</th>
<th>After</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<video src="https://user-images.githubusercontent.com/9600921/162992271-78a9fb21-ad5d-4604-81ad-54592ed11158.mp4" controls="controls" muted="muted" />
</td>
<td>
<video src="https://user-images.githubusercontent.com/9600921/162992330-b88fa1e7-0dcf-4249-aabf-03ff08cf7900.mp4" controls="controls" muted="muted" />
</td>
</tr>
</tbody>
</table>

### 🧪 Testing

<details>
  <summary>Patch file to apply these changes for testing</summary>

```
Index: stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/chat/ChatFragment.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/chat/ChatFragment.kt b/stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/chat/ChatFragment.kt
--- a/stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/chat/ChatFragment.kt	(revision ed43f3f5b9dc71a3582594e87f073198f5565f4f)
+++ b/stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/chat/ChatFragment.kt	(date 1649774960971)
@@ -80,6 +80,8 @@
         initMessagesViewModel()
         initMessageInputViewModel()
         configureBackButtonHandling()
+
+        binding.messageInputView.setCommandsEnabled(false)
     }
 
     override fun onResume() {
```

</details>

1. Apply the patch above which disables commands in the message composer
2. Navigate to the message list screen
3. Select a file in attachment picker
4. Remove the file preview from the message composer
5. Start typing a command
6. Command suggestion popup should not appear

### ☑️Contributor Checklist

#### General

#### Code & documentation

### ☑️Reviewer Checklist

### 🎉 GIF

_Please provide a suitable gif that describes your work on this pull request_
